salesagility/SuiteCRM
# FIXED #7284 - Top of dashlets being cut off by nav bar nd positioning of dashlet pop-up

## Description

The top of dashlets are being cutoff by the navbar on chrome. 



## Motivation and Context

https://github.com/salesagility/SuiteCRM/issues/7284
## How To Test This


1. go to home page
2. edit dashlet on home page
3. scroll up and down to make sure dashlet doesn't disappear behind the nav bar.

Other fix instructions below.

1. Add lots or random dashlets.
2. scroll to the bottom of the home page.
3. click edit on the very bottom dashlet of home page.
4. make sure the dashlet opens in the middle of the screen from the position you are on the screen and DOESN'T throw you back to the top of the screen.

## Types of changes


### Final checklist
